jupyterlab/labbox_ephys_widgets_jp/src/extensions/common/PlotGrid.tsx

```diff
@@ -21,7 +20,7 @@ interface Props {
     hither: HitherContext
 }
 
-const PlotGrid: FunctionComponent<Props> = ({ sorting, onUnitClicked, selections, focus,
+const PlotGrid: FunctionComponent<Props> = ({ sorting, onUnitClicked, selections,
```

Comment:
i am removing the focusedUnitId throughout for now, because it simplifies things... and if we find we need it, we can put it back in. Right now the only thing it was being used for (besides highlighting purposes) was to establish the anchor unit id for shift+click selection. But I now handle that internal to the table, so it doesn't need to be tracked outside of that component.